pkgs/tools/misc/qflipper/default.nix

```diff
@@ -82,20 +87,13 @@ mkDerivation {
   installPhase = ''
     runHook preInstall
 
+    make install
```

Comment:
This is not how it should be done. If you don't overwrite installPhase nix will automatically call make with the right arguments. The other stuff here should then be moved to postInstall.